### PR TITLE
[MIG] easypost_connector: Migrate module dependancies

### PR DESCRIPTION
**Pull request by tedsalmon**
- Migrate tests and dependencies for StockDeliveryRate
- Migrate tests and dependencies for StockDeliveryPackage
- Migrate tests and dependencies for StockDeliveryLabel
- Now using OCA modules from stock.picking

### CODE REVIEW

File: connector_easypost/models/product_packaging.py [hunk 1]
```diff
@@ -208,13 +226,23 @@ def height(self, record):
     @mapping
     @changed_by('weight', 'weight_uom_id')
     def weight(self, record):
```
Comment (tedsalmon) [inline connector_easypost/models/product_packaging.py:228]: Hey @lasley,

Open to ideas here - the product.packaging weight is solely the weight of the packaging, not the packages inside of it. I'm using the picking weight but I'm not sure if there are edge cases where this won't work.

Thanks!

**Comment by tedsalmon** [comment]
FWIW, Here's a picture of it working:

screen shot 2016-10-09 at 23 16 59

File: connector_easypost/models/product_packaging.py (hunk 1 above)
Comment (lasley) [reply connector_easypost/models/product_packaging.py:228]: I'm fine with that. Add to known issues/roadmap in readme

File: connector_easypost/views/stock_picking_package_view.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2016 LasLabs Inc.
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+
+<odoo>
+    <data>
```
Comment (lasley) [inline connector_easypost/views/stock_picking_package_view.xml]: No data tag needed w/ new odoo tag

File: oca_dependencies.txt [hunk 3]
```diff
@@ -1,2 +1,3 @@
 connector
-laslabs-stock https://github.com/laslabs/odoo-stock.git
+stock-logistics-workflow https://github.com/laslabs/stock-logistics-workflow.git feature/9.0/LABS-187-migrate-connectoreasypost-to-oca
```
Comment (lasley) [inline oca_dependencies.txt:2]: Add the fact that this is not using OCA/9.0 to Readme and note the PRs that should be monitored in order to do so.

**Review by lasley — APPROVED** [review]
Minor nitpicks, otherwise LGTM

File: connector_easypost/__openerp__.py [hunk 4]
```diff
@@ -11,8 +11,11 @@
     'license': 'AGPL-3',
     'website': 'https://laslabs.com',
     'depends': [
+        'stock',
         'connector',
-        'stock_delivery_label_new',
+        'stock_picking_rate',
+        'stock_picking_package_info',
+        'base_delivery_carrier_label',
```
Comment (lasley) [inline connector_easypost/__openerp__.py:18]: Where is this provided? The repos/branches defined in OCA dependencies do not contain it

Comment (lasley) [reply connector_easypost/__openerp__.py:18]: The issue is that you need to define the 9.0 branch explicitly. 10.0 is now default.

**Review by lasley — CHANGES_REQUESTED** [review]
nitpicks & oca_depends

**Review by lasley — CHANGES_REQUESTED** [review]
nitpicks & oca_depends

File: connector_easypost/__openerp__.py (hunk 4 above)
Comment (tedsalmon) [reply connector_easypost/__openerp__.py:18]: I have updated connector in oca_dependancies.txt to explicitly require 9.0 for connector. The right branches are already provided for the repos providing `stock_picking_rate`, `stock_picking_package_info` and `base_delivery_carrier_label` 😄

**Comment by tedsalmon** [comment]
Hey @lasley,

The requested changes have been made 😄

**Review by lasley — APPROVED** [review]

**Pull request merged**